lib/models/payment_minutiae.dart

```diff
@@ -97,6 +97,10 @@ class _PaymentMinutiaeFactory {
   }
 
   String _title() {
+    if (_payment.paymentType == PaymentType.Received) {
```

Comment:
@ademar111190  Not sure I understand here. Is it we return "Bitcoin Transfer" for every received payment?

Reply:
True! My bad.
I changed to the second option, which is checking the text itself despite of it being a fragile option.